config/opensearch_dashboards.yml

```diff
@@ -285,4 +285,7 @@
 # opensearchDashboards.survey.url: "https://survey.opensearch.org"
 
 # Set the value of this setting to true to enable plugin augmentation on Dashboard
-# vis_augmenter.pluginAugmentationEnabled: true
\ No newline at end of file
+# vis_augmenter.pluginAugmentationEnabled: true
+
+# Set the value to true enable workspace feature
```

Comment:
nit, just add `to` to the comment :) - Set the value to true **to** enable workspace feature

Reply:
Thanks and done.